### PR TITLE
Fixing python virtualenv

### PR DESCRIPTION
**Pull request by anshikavashistha**

### CODE REVIEW

**Comment by anshikavashistha** [comment]
@Zlopez Have a look on this PR?
I have some fault in my VMware due to which I would not be able to make changes in that current PR.

File: anitya/lib/plugins.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 """Module handling the load/call of the plugins of anitya."""
 
 import logging
-
+import importlib
```
Comment (Zlopez) [inline anitya/lib/plugins.py:22]: This import isn't necessary.

File: createdb.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 # -*- coding: utf-8 -*-
 """createdb"""
 import os
+import importlib
```
Comment (Zlopez) [inline createdb.py:5]: This import isn't necessary.

**Review by Zlopez — CHANGES_REQUESTED** [review]
Some things are still not correct. Also you have a conflict in this PR

**Pull request closed without merging**